src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -81,6 +81,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DelaySign Condition="'$(DelaySign)'==''">false</DelaySign>
     <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)'==''">false</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
+    <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
```

Comment:
I was just talking about this with @jinujoseph, basically the properties that need defaults are those that are not strings (e.g. Booleans). These are the only 2 that I found on this bucket that needed to be default. She tried this change on top of the CPS changes and the visible property changes and things are working as expected (sans other unrelated issues).